source/common/http/http2/codec_impl.cc

```diff
@@ -351,6 +351,13 @@ void ConnectionImpl::StreamImpl::encodeTrailersBase(const HeaderMap& trailers) {
 }
 
 void ConnectionImpl::StreamImpl::encodeMetadata(const MetadataMapVector& metadata_map_vector) {
```

Comment:
should disabling metadata also be supported for decode path?

Reply:
Sure!